sql-parser/dialect/mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLStatementSQLVisitor.java

```diff
@@ -942,7 +949,13 @@ public final ASTNode visitPositionFunction(final PositionFunctionContext ctx) {
     @Override
     public final ASTNode visitSubstringFunction(final SubstringFunctionContext ctx) {
         calculateParameterCount(Collections.singleton(ctx.expr()));
```

Comment:
Can you push your code to this pr? `calculateParameterCount` is still here.

Reply:
Done